pages/common/ncmpcpp.md

```diff
@@ -0,0 +1,19 @@
+# ncmpcpp
+
+> Ncmpcpp is a command line music player.
+
+- Connect to a music player daemon on a given host and port:
+
+`ncmpcpp --host {{ip}} --port {{port}}`
+
+- Print the current song to stdout:
+
+`ncmpcpp --current-song`
+
+- Use specific config:
+
+`ncmpcpp --config {{file}}`
+
+- Use different key bindings:
```

Comment:
Perhaps `Use a different set of key bindings from a file` would read better? I'm not sure here - that doesn't sound quite right either.